daisy/workflow/step_create_disks.go

```diff
@@ -47,11 +47,15 @@ func (c *CreateDisks) validate(w *Workflow) error {
 			return fmt.Errorf("cannot create disk: image not found: %s", cd.SourceImage)
 		}
 
-		_, err := strconv.ParseInt(cd.SizeGB, 10, 64)
-		if err != nil {
+		if _, err := strconv.ParseInt(cd.SizeGB, 10, 64); cd.SizeGB != "" && err != nil {
```

Comment:
technically it will return 0 but only because that's what happens when you declare an int, the error it gives is this: strconv.ParseInt: parsing "": invalid syntax